app/store.js

```diff
@@ -0,0 +1,6 @@
+import {createStore, combineReducers} from 'redux';
+import {channels} from './reducers/channels';
+import {messages} from './reducers/messages';
```

Comment:
I guess it will be better to define  file `reducer.js` in reducers folder. And call function `combineReducers` there. And here just import reducer.